docker/nginx/sites-available/courtlistener.com.conf

```diff
@@ -18,13 +18,18 @@ server {
         expires max;
     }
     # White list of media files we serve directly
-    location ~ ^/(?:doc|docx|html|m4a|mp3|MP3|mp4|nsf|p|pdf|recap-thumbnails|rm|txt|wma|wpd|wsdl)/ {
```

Comment:
I'm not familiar with RM files --- is this correct?

Reply:
Realmedia. I kid you not.